src/Webpack5Cache.js

```diff
@@ -11,7 +11,7 @@ export default class Cache {
   }
 
   isEnabled() {
-    return Boolean(this.compilation.cache);
+    return Boolean(this.compilation.getCache());
```

Comment:
```suggestion
    return true;
```

This always exists and `getCache()` always returns something.

It's not detectable if the cache is enabled. When no memory cache or filesystem cache is attached `cache.get` always returns `undefined` and `store` does nothing. Is this needed to be detectable? @evilebottnawi

Reply:
I will add tests on this, but I think it is unnecessary